playbooks/testdrive.yml

```diff
@@ -36,12 +36,13 @@
     - edxlocal
     - role: mongo
       when: "'localhost' in EDXAPP_MONGO_HOSTS"
+    # Do we relly need edxapp 2x times here?
     - { role: 'edxapp', celery_worker: True }
     - edxapp
-    - notifier
     - analytics_api
     - insights
     - edx_notes_api
+    # Do we really need this 2x times?
```

Comment:
I think so. My impression is that the first `demo` role installs the default Open edX course that's installed with every site deployment. And the second `demo` role is the Appsembler branded demo course. 

@bezidejni might be the best person to ask about this one.